ldml/index.md

```diff
@@ -0,0 +1,46 @@
+
+Like the [Keyman Keyboard Language](https://help.keyman.com/developer/language/) (`.kmn`) format, the LDML Keyboard format defines how keystrokes are interpreted and how keys are laid out in a virtual keyboard.  The LDML format has been defined from the beginning to become a cross-platform industry standard, with active participation from several Unicode member organizations including SIL International.  The intention is to provide a draft specification as a technical preview for public review in mid-2023, with a first release later in 2023.
+
+### Keyman and LDML
+
+As was announced in the [March 2022](https://blog.keyman.com/2022/03/keyman-roadmap-march-2022/) [February 2023](https://blog.keyman.com/2023/02/keyman-roadmap-february-2023/) roadmaps, the Keyman team has been working in anticipation of LDML support in Keyman and Keyman Developer.  We plan to provide initial support for the LDML technical preview version in desktop devices in the Keyman 17.0 release and some supporting tools, with Keyman Developer support and touch platforms coming in version 18.0.
+
+The LDML support is being developed in our cross-platform Keyman Core component, which will enable a consistent feature set across all platforms including web by version 19.0, with a common code base.
+
+![Movie: Keyman on Linux, typing using an LDML keyboard](/cdn/dev/img/ldml-keyman-linux.gif)
+
+## Frequently Asked Questions for Keyman LDML Support
+
+
+- *How will the LDML format be supported with Keyman?*
+
+  From a technical perspective, the LDML format will be a newly supported input to the Keyman Command Line Compiler. The XML file will be compiled into a <a href="https://help.keyman.com/developer/16.0/reference/file-types/kmx">KMX compiled keyboard file</a>. The .kmx file will be read and used by the Keyman Core component, starting in version 17.0</dd>
+
+- *How will the LDML format be supported with Keyman Developer?*
+
+    LDML support in Keyman Developer is still in the planning stages, but is being planned for Keyman 18.0
+
+- *What is the impact on existing developed keyboards in .kmn format?*
+
+  None. Keyman and Keyman Developer continues to support the thousands of languages in .kmn files and the .kmn format.
+
+- *Will there be a facility available to automatically convert .kmn to LDML or vice versa?*
+
+  The LDML specification is designed primarily to support new keyboard layouts. At a later stage, we would like to make available software to assist developers in migrating keyboards to the new format, including tooling to compare and verify the capabilities of a .kmn versus a similar LDML keyboard definition.</dd>
```

Comment:
Do you still need the `</dd>` tags? Also at the end of l. 29.

fwiw, the Markdown on the site is rendering the italics as bullet points 😞 
![image](https://user-images.githubusercontent.com/7358010/220221124-90da9f65-6e9b-4386-a202-95dd970480ac.png)

Reply:
fixed the dd tag.

Actually the bullet points are there, there's a hyphen at the beginning of each line.  It looks like rather it's ignoring the italics.  That's… fine I suppose.

Reply:
> It looks like rather it's ignoring the italics.

This is due to a very old, less-than-wonderful CSS reset in the design of the site. One day, we'll rewrite the styles for keyman.com to be more modern, and that will fix things like this which annoy me. But lowpri :sad:

Reply:
i've updated it to `**bold**`